Sources/BraveVPN/BraveVPNPreferences.swift

```diff
@@ -28,5 +28,6 @@ extension Preferences {
       static let vpnHostDisplayName = Option<String?>(key: "vpn.host-location", default: nil)
       public static let skusCredential = Option<String?>(key: "skus.credential", default: nil)
       public static let skusCredentialDomain = Option<String?>(key: "skus.credential-domain", default: nil)
+      static let wireGuardProtocolMigrated = Option<Bool>(key: "vpn.migrate.protocol-wireguard", default: false)
```

Comment:
Still needed?

Reply:
No it is not you are right. Fix at https://github.com/brave/brave-ios/pull/7125/commits/97e33c63bf3090065d21db883d2dd08c78f9ed27